packages/venia-concept/src/components/AppShell/appShell.js

```diff
@@ -45,4 +68,14 @@ class AppShell extends Component {
     }
 }
 
-export default classify(defaultClasses)(AppShell);
+const mapStateToProps = ({ app }) => {
+    const { isOnline } = app;
+    return {
+        isOnline
+    };
+};
+
+export default connect(
```

Comment:
When merge conflicts are resolved and these changes ported to `app.js`, make sure to migrate the redux connections to `App/container.js`.